.travis.yml

```diff
@@ -1,20 +1,97 @@
 language: java
-jdk: openjdk8
 dist: trusty
+jdk:
+  - oraclejdk8
+  - oraclejdk9
+  - oraclejdk11
+  - openjdk8
 
 #enable caching
 before_cache:
   - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
   - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+
 cache:
   directories:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper/
-    
+    - $HOME/ibm-jdk-installer/
+
+install: true
 script:
-  - wget https://raw.githubusercontent.com/sormuras/bach/76983e2f99497738b6f3e9f86bd42fd3a759a275/install-jdk.sh -O $TRAVIS_BUILD_DIR/install-jdk.sh
-  - bash $TRAVIS_BUILD_DIR/install-jdk.sh --feature 11 -L GPL --target $HOME/openjdk11
-  - ./gradlew build codeCoverageReport -PsystemTestJavaHomes="$JAVA_HOME:$HOME/openjdk11"
-after_success:
-  # upload coverage report
-  - bash <(curl -s https://codecov.io/bash)
\ No newline at end of file
+  - ./gradlew :inspectit-oce-agent:systemTest
+
+jobs:
+  include:
+    - name: "JDK ibmjdk8 Java"
+      sudo: true
+      install:
+        - wget -nc -O $HOME/ibm-jdk-installer/ibm-sdk.bin https://s3-eu-west-1.amazonaws.com/inspectit-oce-files/ibm-java-sdk-8.0-5.26-x86_64-archive.bin || true
```

Comment:
the build is failing again, I guess this is because of the GC test, can we please merge #43 first then and then prove that this is green?

Reply:
Yes, that's the reason